### PR TITLE
4.1-pre-release 

### PR DESCRIPTION
**Pull request by lk86**
Tighten up flags to daemon in default charts, rename package to mina, prepare for tagging / branch cut. Also updates coda-automation to match master w/ the latest docker image, genesis ledger, etc. to use for the community qa net.

Additionally, fixes the zexe-standardize.sh script for the new marlin rust bindings, as well as making bisect_ppx conditional for the new rust bindings to avoid spamming ~/ with .coverage files.

Also adds the changes to rebuild-deb that included the last generate-keypair package manifest so that it can be rebuilt in CI more easily + to rename to mina like the other packages.

### CODE REVIEW

**Comment by psteckler** [comment]
As I mentioned to @lk86, I have a branch that uses the latest `bisect_ppx`, and eliminates the need to mention `--conditional`. The ppx will be mentioned in an `instrumentation` clause instead of in the `pps` clause.

It's a big change, and it's low priority, so it may take some time before I make it into a PR.

File: dockerfiles/Dockerfile-rosetta [hunk 1]
```diff
@@ -248,6 +248,9 @@ RUN git pull \
    && go build -o ${HOME}/app/libp2p_helper \
    && go clean --cache --modcache --testcache -r
 
+# HACK: build without special cpu features to allow more people to run coda
+RUN sed -i 's/+bmi2,+adx/-bmi2,-adx/g' src/lib/zexe/snarky-bn382/dune
```
Comment (mrmr1993) [inline dockerfiles/Dockerfile-rosetta:252]: This probably wants to patch `src/lib/marlin_plonk_bindings/stubs/dune` as well.

**Review by mrmr1993 — APPROVED** [review]
The script in `scripts/archive/build-release-archives.sh` probably wants updating too. Other than that and my other comment, LGTM.

**Review by nholland94 — APPROVED** [review]

File: scripts/publish-deb.sh [hunk 2]
```diff
@@ -12,7 +12,7 @@ DEBS3='deb-s3 upload '\
 '--s3-region=us-west-2 '\
 '--bucket packages.o1test.net '\
 '--preserve-versions '\
-'--lock '\
+'--no-lock '\
```
Comment (mrmr1993) [inline scripts/publish-deb.sh:15]: NOOOOOOO! Delete the lockfile as described in the comment above, don't remove the lock flag!!

**Review by mrmr1993 — CHANGES_REQUESTED** [review]

**Review by mrmr1993 — APPROVED** [review]

**Pull request merged**